relay-client/src/lib.rs

```diff
@@ -0,0 +1,172 @@
+//! Orb-Relay crate
+use orb_relay_messages::{common, prost::Name, prost_types::Any, self_serve};
+
+pub mod client;
+
+#[allow(missing_docs)]
```

Comment:
```suggestion
#[expect(missing_docs)]
```

lets change all the `allow`s to `expect`s

Reply:
Removed for this PR. Refactored in PR #342